test/test-execute/exec-inaccessiblepaths-proc.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Test to make sure that mount namespace setup works properly with the 'InaccessiblePaths=/proc' option
+
+[Service]
+InaccessiblePaths=/proc
+ExecStart=/bin/sh -x -c 'test ! "$(ls -A /proc)"'
```

Comment:
Thank you for adding the test.

I'm not sure I understand why we need `-A`. Besides, I think the check is a bit complicated. Probably we should simply check that the access rights are `0`.

Talking of running `test-execute` by the CI, https://anonscm.debian.org/cgit/pkg-systemd/systemd.git/tree/debian/tests/root-unittests runs it.